.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: push
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [10.x, 12.x]
+    steps:
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: actions/checkout@v2
+      - run: npm install -g yarn
```

Comment:
Judging from davids PR in the timepicker (https://github.com/COMPEON/timepicker/pull/24/files), this is not needed.